### ISSUE TITLE
Unable to initialize tonlib object in Android project

### ISSUE THREAD

**Issue by Good-rgb** [neodix42/ton4j#71]
1、I added JSON
val configStr = Utils.streamToString(openJson)
val build = Tonlib.builder().testnet(true).originalGlobalConfigStr(configStr).build()

2、error log 
Error creating tonlib instance: Attempt to invoke virtual method 'org.ton.java.tonlib.types.globalconfig.LiteServers[] org.ton.java.tonlib.types.globalconfig.TonGlobalConfig.getLiteservers()' on a null object reference

**Comment by neodix42** [comment]
could you please show the value of configStr just before feeding to the Tonlib builder?

**Comment by Good-rgb** [comment]
Yes, the value of configStr that I printed out has JSON and server node configuration, but initializing Tonlib will result in an error
1、Full Code
val open = assets.open("testnet-global-config.json")
 val configStr = Utils.streamToString(open)
val build = Tonlib.builder().testnet(true).originalGlobalConfigStr(configStr).build()

2、This is the path and content of the JSON file
<img width="933" alt="image" src="https://github.com/user-attachments/assets/993f9cda-5293-4eaa-b918-eca51dfde776">

<img width="341" alt="image" src="https://github.com/user-attachments/assets/00e6907a-8af0-4670-ab59-8dd730c59d11">

**Comment by neodix42** [comment]
Thanks for the details. I will try to reproduce the problem an be back to you.

**Comment by Good-rgb** [comment]
Now the above problem does not occur, but the following problem occurs:
Error creating tonlib instance: Native library (com/sun/jna/android-aarch64/libjnidispatch.so) not found in resource path (.)
And I still get an error after importing so. When will you release an optimized version for Android?

**Comment by Good-rgb** [comment]
<img width="1720" alt="Image" src="https://github.com/user-attachments/assets/91610278-97a4-4897-b715-31732c700315" />
